### DISCUSSION TITLE
Codespaces usage increase but not used

### DISCUSSION THREAD

**Discussion by Glydric** [community/community#59899] [Codespaces]
### Select Topic Area

Bug

### Body

Hi, i receive the codespace warning for resource usage but in that account (the only i have) i have no machines

**Comment by samruddhikhandale** [comment]
Hi 👋 

Looks like you have prebuilds enabled which are consuming the resources.
Look at [about-billing-for-github-codespaces#storage-costs-for-prebuilds](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#storage-costs-for-prebuilds) and [about-billing-for-github-codespaces#controlling-the-cost-of-prebuilds](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#controlling-the-cost-of-prebuilds) for more information.

Let us know if you have any other questions, thanks!
